CONTRIBUTING.md

```diff
@@ -40,18 +39,46 @@ DB_HOST=db
 DB_PREFIX=na
 ```
 
+## Useful `make` commands
```

Comment:
`make help` is also useful

```
[~/git/bmlt-enabled/bmlt-root-server]$ make help
bash                           Runs bash shell in apache container
clean                          Clean build
composer                       Runs composer install
coverage-serve                 Serves HTML Coverage Report
coverage                       Generates HTML Coverage Report
crouton                        Installs crouton
dev                            Docker Compose Up
docker-publish-base            Builds Base Docker Image
docker-push                    Pushes docker image to Dockerhub
docker                         Builds Docker Image
help                           Print the help documentation
lint-fix                       PHP Lint Fix
lint                           PHP Lint
mysql                          Runs mysql cli in mysql container
test                           Runs PHP Tests
zip                            Builds zip file
[~/git/bmlt-enabled/bmlt-root-server]$ 

```

Reply:
The `make clean` target really removes all build artifacts and downloaded dependencies. Not sure how to say that succinctly.

You really only need to do `make docker` when first getting set up and after a change has been made to the Dockerfile or its base image.

Reply:
Added these.